### DISCUSSION TITLE
What is StreamFX using when "Resize Output" is selected for the NVENC codec?

### DISCUSSION THREAD

**Discussion by jukofyork** [Xaymar/obs-stream-effects#980] [Q&A] [answered]
Just looking through the code and trying to work out what scaling code is invoked when using the "Resize Output" option with StreamFX's NVENC codec?

I can see ffmpeg_instance::initialize_sw() has a call using libswscale's "SWS_SINC" filter, but this seems to only be for the "AVMEDIA_TYPE_VIDEO" codec.

Also, do you have any idea what stock OBS is using when the "Resize Output" option is selected? I can see a call using libswscale's "SWS_POINT" filter but again not 100% sure if this is just some fallback option for certain codec(s).

**Comment by Xaymar** [comment, marked as answer]
If you're talking about which scaling method is used:

- Sinc with full chroma and high accuracy for Software mode, i.e. enabling Resize Output.
- Whatever you selected in the Video tab for Hardware mode. 

For OBS Studio, please ask in their own forums and messaging tools.
